ttools/callbacks.py

```diff
@@ -460,14 +460,23 @@ class CheckpointingCallback(Callback):
 
     PERIODIC_PREFIX = "periodic_"
     EPOCH_PREFIX = "epoch_"
+    BEST_MODEL_FILENAME = "best"
 
     def __init__(self, checkpointer, interval=600,
-                 max_files=5, max_epochs=10):
+                 max_files=5, max_epochs=10,
+                 best_val_key=None, best_val_value=None):
```

Comment:
do we need to initialize best_val_value externally? could that be handled by the load/save mechanism so we only have one parameter?